- XXX no comment
- Implement support for PT_OPENBSD_NOBTCFI in lld(1).  This can be set using
  the -z nobtcfi option.

Index: lld/ELF/Config.h
--- lld/ELF/Config.h.orig
+++ lld/ELF/Config.h
@@ -236,6 +236,7 @@ struct Configuration {
   bool zInitfirst;
   bool zInterpose;
   bool zKeepTextSectionPrefix;
+  bool zNoBtCfi;
   bool zNodefaultlib;
   bool zNodelete;
   bool zNodlopen;
@@ -266,8 +267,13 @@ struct Configuration {
   ELFKind ekind = ELFNoneKind;
   uint16_t emachine = llvm::ELF::EM_NONE;
   llvm::Optional<uint64_t> imageBase;
+  // commonPageSize and maxPageSize are influenced by nmagic or omagic
+  // so may be set to 1 if either of those options is given.
   uint64_t commonPageSize;
   uint64_t maxPageSize;
+  // textAlignPageSize is the target max page size for the purpose
+  // of aligning text sections, which may be unaligned if given nmagic
+  uint64_t textAlignPageSize;
   uint64_t mipsGotSize;
   uint64_t zStackSize;
   unsigned ltoPartitions;
